src/controllers/country.controller.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { Response, Request } from 'express';
+import * as contryService from '../services/country.service';
+import { getPhoneNumberByCountry } from '../services/phoneNumber.service';
+
+export const getPhoneNumbers = async(req: Request, res: Response) => {
+  try {
+    const country_code = req.params.code;
```

Comment:
Mình đặt tên biến theo kiểu camelCase chị
```suggestion
    const countryCode = req.params.code;
```